docs/data/material/migration/migration-v4/v5-style-changes.md

```diff
@@ -0,0 +1,695 @@
+
+```diff
+-import { fade } from '@mui/material/styles';
++import { alpha } from '@mui/material/styles';
+
+  const classes = makeStyles(theme => ({
+-  backgroundColor: fade(theme.palette.primary.main, theme.palette.action.selectedOpacity),
++  backgroundColor: alpha(theme.palette.primary.main, theme.palette.action.selectedOpacity),
+  }));
+```
+
+### ✅ Update createStyles import
+
+The `createStyles` function from `@mui/material/styles` was moved to the one exported from `@mui/styles`. It is necessary for removing the dependency to `@mui/styles` in the core package.
+
+```diff
+-import { createStyles } from '@mui/material/styles';
++import { createStyles } from '@mui/styles';
+```
+
+### ✅ Update createGenerateClassName import
+
+The `createGenerateClassName` function is no longer exported from `@mui/material/styles`. You should import it directly from `@mui/styles`.
+
+```diff
+-import { createGenerateClassName } from '@mui/material/styles';
++import { createGenerateClassName } from '@mui/styles';
+```
+
+To generate custom class names without using `@mui/styles`, check out [ClassName Generator](/material-ui/experimental-api/classname-generator/) for more details.
+
+### ✅ Rename createMuiTheme
+
+The function `createMuiTheme` was renamed to `createTheme` to make it more intuitive to use with `ThemeProvider`.
+
+```diff
+-import { createMuiTheme } from '@mui/material/styles';
++import { createTheme } from '@mui/material/styles';
+
+-const theme = createMuiTheme({
++const theme = createTheme({
+```
+
+### ✅ Update jssPreset import
+
+The `jssPreset` object is no longer exported from `@mui/material/styles`. You should import it directly from `@mui/styles`.
+
+```diff
+-import { jssPreset } from '@mui/material/styles';
++import { jssPreset } from '@mui/styles';
+```
+
+### ✅ Update `makeStyles` import
+
+The `makeStyles` JSS utility is no longer exported from `@mui/material/styles`.
+You can use `@mui/styles/makeStyles` instead.
+
+Make sure to add a `ThemeProvider` at the root of your application, as the `defaultTheme` is no longer available.
+
+If you are using this utility together with `@mui/material`, it's recommended that you use the `ThemeProvider` component from `@mui/material/styles` instead.
+
+```diff
+-import { makeStyles } from '@mui/material/styles';
++import { makeStyles } from '@mui/styles';
++import { createTheme, ThemeProvider } from '@mui/material/styles';
+
++const theme = createTheme();
+  const useStyles = makeStyles((theme) => ({
+    background: theme.palette.primary.main,
+  }));
+  function Component() {
+    const classes = useStyles();
+    return <div className={classes.root} />
+  }
+
+  // In the root of your app
+  function App(props) {
+-  return <Component />;
++  return <ThemeProvider theme={theme}><Component {...props} /></ThemeProvider>;
+  }
+```
+
+### ✅ Update MuiThemeProvider import
```

Comment:
I would suggest moving this diff up, to be right after `✅ Rename createMuiTheme` the two changes make sense together.